python/pyspark/sql/tests/pandas/test_pandas_udf_grouped_agg.py

```diff
@@ -475,7 +475,7 @@ def test_invalid_args(self):
         mean_udf = self.pandas_agg_mean_udf
 
         with QuietTest(self.sc):
-            with self.assertRaisesRegex(AnalysisException, "nor.*aggregate function"):
+            with self.assertRaisesRegex(AnalysisException, "[MISSING_AGGREGATION]"):
```

Comment:
We should test this with similar logic with `checkError` in Scala side.

Let me create the JIRA and update soon after getting some feedback from community.